src/components/SearchGridManualLoad.vue

```diff
@@ -149,6 +149,7 @@ export default {
 
 <!-- Add "scoped" attribute to limit CSS to this component only -->
 <style lang="scss" scoped>
+  @import "node_modules/bulma/sass/utilities/_all";
```

Comment:
I think importing only `mixins.sass` should be enough. I don't think you need to import `_all` https://github.com/jgthms/bulma/blob/master/sass/utilities/mixins.sass

Reply:
I did that but mixins.sass depends in other file and that other file depends on another one so I thought it will be better just import _all.sass instead of three separated files. Should I rather import the three files?

Reply:
we can also import like this also, 

`<style lang="sass" src="bulma"></style>`
`<style>
  /* Your css for this file... */
</style>`
source: [stack-overflow](https://stackoverflow.com/questions/41966491/css-frameworks-in-vue-js)
 bulma's main entry in package.json points to `bulma.sass`  

I think it looks better like this! Just a suggestion.

Reply:
@brenoferreira If we are just importing 
`@import 'bulma/sass/utilities/mixins';`
then we need to be careful regarding the syntax I guess because 

in sass it should be
`+desktop
color: red`

in scss
`@include desktop {
color: red;
}`

Source : [Issue](https://github.com/jgthms/bulma/issues/351)